### PR TITLE
Remove ANALYZED / FORECAST concept

### PR DESCRIPTION
**Pull request by joakim-hove**
This PR is a step in the long running series: "Remove support for running EnKF".

### CODE REVIEW

File: devel/libenkf/include/ert/enkf/fs_types.h [hunk 1]
```diff
@@ -63,7 +63,10 @@ typedef enum {
   DRIVER_STATIC           = 2,  // Driver static is no longer in use since December 2015 - but it must be retained here for old mount files on disk.
   DRIVER_INDEX            = 4,  // DRIVER_DYNAMIC = 3; removed at svn ~ 3720.
   DRIVER_DYNAMIC_FORECAST = 5,
-  DRIVER_DYNAMIC_ANALYZED = 6} fs_driver_enum;
+  DRIVER_DYNAMIC_ANALYZED = 6   // Driver DYNAMIC_ANALYZED is no longer in use since April 2015 - but it must be retained here for old mount files on disk.
```
Comment (jepebe) [inline devel/libenkf/include/ert/enkf/fs_types.h]: April 2016?

File: devel/python/python/ert_gui/tools/export/export_panel.py [hunk 2]
```diff
@@ -126,15 +125,10 @@ def export(self):
         path = self.__file_name.text()
         iactive = self.__active_realizations_model.getActiveRealizationsMask()
         file_type_key = self.__file_type_model[self.__file_type_combo.currentIndex()]
-        state = self.getEnkfStateType(keyword)
-        values = {"keyword":keyword, "report_step":report_step, "iactive":iactive,"file_type_key":file_type_key, "state":state, "selected_case":selected_case, "path":path}
+        values = {"keyword":keyword, "report_step":report_step, "iactive":iactive,"file_type_key":file_type_key, "path":path}
```
Comment (jepebe) [inline devel/python/python/ert_gui/tools/export/export_panel.py]: Is there a reason for removing `selected_case` as well?

File: devel/libenkf/include/ert/enkf/enkf_state.h [hunk 3]
```diff
@@ -64,7 +64,7 @@ typedef struct enkf_state_struct    enkf_state_type;
   //void             * enkf_state_complete_forward_model__(void * arg );
   void *             enkf_state_load_from_forward_model_mt( void * arg );
   void               enkf_state_initialize(enkf_state_type * enkf_state , enkf_fs_type * fs, const stringlist_type * param_list , init_mode_type init_mode);
-  void               enkf_state_fread(enkf_state_type *  , enkf_fs_type * fs , int  , int  , state_enum );
+  void               enkf_state_fread(enkf_state_type *  , enkf_fs_type * fs , int  , int );
   bool               enkf_state_get_analyzed(const enkf_state_type * );
```
Comment (jepebe) [inline devel/libenkf/include/ert/enkf/enkf_state.h]: This should probably be removed?

**Comment by jepebe** [comment]
Found commented `state_enum` in `ert_run_context.c`
Found reference in `tips.txt`

**Comment by jepebe** [comment]
`export_misfit_data.py` refers to `EnkfStateType`

**Comment by jepebe** [comment]
"forecasted" is mentioned in `ENKF_TRUNCATION` in `keywords/index.rst`

File: devel/python/python/ert_gui/tools/export/export_panel.py (hunk 2 above)
Comment (joakim-hove) [inline devel/python/python/ert_gui/tools/export/export_panel.py]: No - that was a bug.

**Comment by joakim-hove** [comment]
> "forecasted" is mentioned in ENKF_TRUNCATION in keywords/index.rst

That is OK - otherwise all comments have been addressed in rebase & push.

**Pull request merged**